awspec.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'addressable'
   spec.add_runtime_dependency 'awsecrets', '~> 1'
   spec.add_runtime_dependency 'aws-sdk', '~> 3'
-  spec.add_runtime_dependency 'dry-inflector'
+  spec.add_runtime_dependency 'dry-inflector', '>= 1.1'
```

Comment:
Instead of requiring `>= 1.1` we could require `>= 0.1.2`, (keeping support for older ruby versions), and then replace instances of `Dry::Inflector.new` in https://github.com/k1LoW/awspec/blob/7c19ade77e372bcc4a74bdfbbca85c030dcd1377/lib/awspec/ext/string.rb with

```ruby
Dry::Inflector.new { |inflections| inflections.acronym 'DB' }
```

It would add `DB` as an acronym to the older versions that didn't have it as a default.  (Versions older than `0.1.2` don't support doing this.

Reply:
Unfortunately, there's no mechanism to remove a default as discussed here. https://github.com/dry-rb/dry-inflector/issues/44